db/migrate/20131030184435_create_houses.rb

```diff
@@ -1,10 +1,9 @@
 class CreateHouses < ActiveRecord::Migration
   def change
     create_table :houses do |t|
-      f.string :name, null: false
-      f.string :img_url
-
-      f.timestamps
+      t.string :name, null: false
+      t.string :image_url
```

Comment:
You didn't need to change the column name, but as long as you updated other references to this attribute then you're good.